postman/Get Broadcast Details.postman_collection.json

```diff
@@ -0,0 +1,549 @@
+							"    if (pm.expect(jsonData.entries.length).is.above(0)) {",
+							"        pm.globals.set(\"draft_broadcast_link\", jsonData.entries[0].self_link);",
+							"    }",
+							"});",
+							"",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/broadcasts?status=draft",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"broadcasts"
+					],
+					"query": [
+						{
+							"key": "status",
+							"value": "draft"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get First Draft Broadcast",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{draft_broadcast_link}}",
+					"host": [
+						"{{draft_broadcast_link}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Scheduled Broadcasts",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "bdc9f89d-300b-424d-8f3e-99b24d49ec71",
+						"exec": [
+							"pm.test(\"Successfully retrieved link for first scheduled broadcast\", function(){",
+							"    var jsonData = pm.response.json();",
+							"    if (pm.expect(jsonData.entries.length).is.above(0)) {",
+							"        pm.globals.set(\"scheduled_broadcast_link\", jsonData.entries[0].self_link);",
+							"    }",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/broadcasts?status=scheduled",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"broadcasts"
+					],
+					"query": [
+						{
+							"key": "status",
+							"value": "scheduled"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get First Scheduled Broadcast",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{scheduled_broadcast_link}}",
+					"host": [
+						"{{scheduled_broadcast_link}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Sent Broadcasts",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "42349437-b50b-47da-8b97-d530b3aeffe7",
+						"exec": [
+							"pm.test(\"Successfully retrieved link for first sent broadcast\", function(){",
+							"    var jsonData = pm.response.json();",
+							"    if (pm.expect(jsonData.entries.length).is.above(0)) {",
```

Comment:
You don't need to check the return value here. The assertion will stop the test if it fails.

Reply:
This is still checking the return value